tests/functional/list/test_list.py

```diff
@@ -724,12 +710,12 @@ def expect_resource_type_env_var(self):
         }
         del os.environ["DBT_EXCLUDE_RESOURCE_TYPES"]
 
-    def expect_selected_keys(self, project):
+    def expect_selected_keys(self, happy_path_project):  # noqa: F811
```

Comment:
In this test we're actually using the happy_path_project fixture argument in the function. Is the # noqa: F811 necessary for another reason?